app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -48,6 +53,7 @@ class HomeScreenRobot {
     fun verifyHomeWordmark() = assertHomeWordmark()
     fun verifyHomeToolbar() = assertHomeToolbar()
     fun verifyHomeComponent() = assertHomeComponent()
+    fun verifyDefaultSearchEngineIsDDG() = verifyDDGSearchEngineIcon()
```

Comment:
I think we should pass in a search engine to verify a default. The default could change in product and we would have to rewrite this function.